teste/SME.SGP.TesteIntegracao/ServicosFakes/ServicoEOLFake.cs

```diff
@@ -592,9 +592,194 @@ public Task<IEnumerable<DisciplinaResposta>> ObterDisciplinasPorCodigoTurmaLogin
             throw new NotImplementedException();
         }
 
-        public Task<IEnumerable<DisciplinaDto>> ObterDisciplinasPorIdsAgrupadas(long[] ids)
+        public async Task<IEnumerable<DisciplinaDto>> ObterDisciplinasPorIdsAgrupadas(long[] ids, string codigoTurma = null)
```

Comment:
se o async for utilizado na assinatura de um método que não chame o await vai gerar um Warning